### PR TITLE
Add view template for homepage

### PR DESCRIPTION
**Pull request by lars3016**
screen shot 2017-03-29 at 17 38 14

### CODE REVIEW

File: app/views/pages/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,32 @@
...
+                 </div>
+             </div>
+         </div>
+
+     </div>
+ </div>
+
+ <script src="js/jquery.js"></script>
+
+ <script src="js/bootstrap.min.js"></script>
```
Comment (miriamtocino) [inline app/views/pages/home.html.erb]: These two scripts don't belong in there.

I have the feeling that you are not following the instructions to setup bootstrap in your project as in the Reader.
Please review [this article](https://reader.codaisseur.com/courses/intermediate-bootcamp-1d45fcf7-b918-482f-a526-e126d3c063d9/57/codaisseurbnb-project-setup/installing-bootstrap).

If you do all the steps in there, you will have a proper application working with bootstrap 😄

File: spec/features/home.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    end
+
+    describe "link HOST TOC" do
+      it "links to /sponsor" do
+        visit root_url
+        click_on "HOST TOC"
+        expect visit "/sponsor"
+      end
+    end
+end
```
Comment (miriamtocino) [inline spec/features/home.html.erb_spec.rb]: Beautiful!

It looks to me that there is also sth wrong with your indentation.

[](http://cd.sseu.re/20170330-j9uqw.png)

File: spec/views/pages/home.html.erb_spec.rb [hunk 3]
```diff
@@ -4,7 +4,8 @@
   describe "rendering home page" do
     it "contains \"Pages#home\"" do
       render
-      response.should have_text("Pages#home")
+        response.should have_text("Taste of code")
+        response.should have_text("Beginner coding workshop")
```
Comment (miriamtocino) [inline spec/views/pages/home.html.erb_spec.rb]: Indentation of these two lines ^^ should be two spaces less, aligned with `render` 😉

File: app/views/pages/home.html.erb (hunk 1 above)
Comment (lars3016) [inline app/views/pages/home.html.erb]: ok thnx I will do this

**Comment by miriamtocino** [comment]
:shipit: !!!

**Pull request merged**